tests/unit/getTestResultsFilters.unitTest.ts

```diff
@@ -12,6 +12,7 @@ describe("TestDataProvider Util functions", () => {
           param1: "something else"
         }
       ];
+      // @ts-ignore
```

Comment:
You needed these ignores for the types? Wouldn't it be better to add meaningful objects?

Reply:
The ignore is for the private function. Call private functions externally is not allowed in TypeScript. JavaScript doesn't really care about such identifiers hence ignore to run tests..

Reply:
Ah sorry, my bad. Didn't notice it was private